common/dev/xdpe12284c.c

```diff
@@ -0,0 +1,131 @@
+#include <stdio.h>
+#include <string.h>
+#include "sensor.h"
+#include "hal_i2c.h"
+#include "pmbus.h"
+#include "util_pmbus.h"
+
+enum {
+	VR12 = 1,
+	VR13,
+	IMVP9,
+};
+
+enum {
+	VID_IDENTIFIER = 1,
+};
+
+/*  Reference: Infineon spec section 8.24: VID table
+ *  PMBUS spec section 8.2: VOUT mode
+ */
+static float vid_to_float(int val, uint8_t vout_mode) {
+	uint8_t mode = 0;
+	uint8_t parameter = 0;
+
+	//VID 0 is always 0 V
+	if (val == 0) {
+		return 0;
+	}
+
+	mode = (vout_mode >> 5);
+
+	if (mode != VID_IDENTIFIER) {
+		printf("%s Infineon VR reading with invalid VID IDENTIFIER: %x", __func__, mode);
+		return -1;
+	}
+
+	switch (vout_mode & 0x1f) {
+	case VR12:
+		if(val > 0) {
+			return ((val - 1) * 5 + 250);
+		}
+	case VR13:
+		if(val > 0) {
+			return ((val - 1) * 10 + 500);
+		}
+	case IMVP9:
+		if(val > 0) {
+			return ((val - 1) * 10 + 200);
+		}
```

Comment:
Add `break;` for all cases

Reply:
Hi @GoldenBug,

Force update the commit by your suggestion, please help review it again, thanks.

Regards,
Jerry